requirements.txt

```diff
@@ -3,5 +3,5 @@ rednose
 cryptography
 pyasn1
 pymodbus
-cpppo
+cpppo==4.3.4
```

Comment:
I don't recall fully why we fixed the version here. Let's test later if this is still required, or this was python2 related